R/convertFlowCore.R

```diff
@@ -0,0 +1,103 @@
+library("stats")
+
+#' Convert a gate to flowCore
+#'
+#' Converts a CellEngine gate ("PolygonGate", "EllipseGate", or "RectangleGate") to
+#' its flowCore analogue.
+#'
+#' @param gate The CellEngine gate to be converted
+convertToFlowCore <- function (gate) {
+  if (!requireNamespace("flowCore")) {
+    message("These utilities require the 'flowCore' package.")
+    return(invisible())
```

Comment:
Would `stop("These utilities require the 'flowCore' package.")` be more appropriate here?